docs/reference/docs/reindex.asciidoc

```diff
@@ -397,7 +397,7 @@ POST _reindex
 {
   "source": {
     "remote": {
-      "host": "http://otherhost:9200",
+      "host": "https://otherhost:9200",
```

Comment:
This isn't going to work with Elasticsearch without x-pack installed. I'd prefer to stick with http. I'd be ok with a callout here mentioning that we also support https and it is required for clusters that use tls with x-pack.